src/components/Autocomplete/components/ComboBox/ComboBox.tsx

```diff
@@ -263,7 +263,7 @@ export default class ComboBox extends React.PureComponent<Props, State> {
         aria-haspopup
         onFocus={this.handleFocus}
         onBlur={this.handleBlur}
-        tabIndex={0}
+        tabIndex={-1}
```

Comment:
As far as I can tell we don't need the tab index on the `div` at all

```suggestion
```

Reply:
Originally I removed the tabIndex entirely, however it failed CircleCI tests that check if an item has an onFocus method, it must have a tabIndex.

Reply:
Hmm. The container shouldn't need to get focus because the text input does naturally. I wonder why `onFocus`/`onBlur` is being handled on the `<div>` and not the field?